launchdarkly/manifest.json

```diff
@@ -1,30 +1,59 @@
 {
-  "integration_id": "launchdarkly",
-  "supported_os": [
-    "linux",
-    "mac_os",
-    "windows"
+  "manifest_version": "2.0.0",
+  "app_uuid": "7144d0c5-42f2-4cc5-b562-5f77debc0c52",
+  "app_id": "launchdarkly",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Configuration Deployment",
```

Comment:
```suggestion
```
Hi @ldhenry, apologies., could we remove this category for now. We will get this added after Dash. Thanks